docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -2,354 +2,336 @@
+
+```typescript
+function OwnedGame({
+  game,
+  trophy,
+  invalidateGame,
+  invalidateTrophy,
+}: {
+  game: GameData;
+  trophy: Trophy;
+  invalidateGame: InvalidateGameQuery;
+  invalidateTrophy: InvalidateTrophyQuery;
+}): ReactElement {
+  const adminKey = game.admin ? new MultiSigPublicKey(new Uint8Array(game.admin)) : null;
+
+  const client = useSuiClient();
+  const signAndExecute = useExecutor();
+  const multiSignAndExecute = useExecutor({
+    execute: ({ bytes, signature }) => {
+      const multiSig = adminKey!!.combinePartialSignatures([signature]);
+      return client.executeTransactionBlock({
+        transactionBlock: bytes,
+        signature: [multiSig, signature],
+        options: {
+          showRawEffects: true,
+        },
+      });
+    },
+  });
+
+  const [turnCap, invalidateTurnCap] = useTurnCapQuery(game.id);
+  const account = useCurrentAccount();
+  const tx = useTransactions()!!;
+
+  // ...
+
+  const onMove = (row: number, col: number) => {
+    signAndExecute(
+      {
+        tx: tx.sendMark(turnCap?.data!!, row, col),
+        options: { showObjectChanges: true },
+      },
+      ({ objectChanges }) => {
+        const mark = objectChanges?.find(
+          (c) => c.type === 'created' && c.objectType.endsWith('::Mark'),
         );
 
-        mark.delete();
-        * &mut game.finished = finished;
-        return
-    } else if (game.cur_turn >= 8) {    // Draw
-        make.delete();
-        * &mut game.finished = 3;
-        return
-    };
+        if (mark && mark.type === 'created') {
+          const recv = tx.receiveMark(game, mark);
+          recv.setSender(adminKey!!.toSuiAddress());
+          recv.setGasOwner(account?.address!!);
+
+          multiSignAndExecute({ tx: recv }, () => {
+            invalidateGame();
+            invalidateTrophy();
+            invalidateTurnCap();
+          });
+        }
+      },
+    );
+  };
 
-    // Next turn
-    * &mut game.cur_turn = game.cur_turn + 1;
-    addr = game.get_cur_turn_address();
-    mark.during_turn = true;
-    transfer::transfer(mark, addr);
+  // ...
 }
 ```
 
-The first function is straightforward. The player sends the location of the mark to the multisig account. Then in the next function, the multisig actually places down the mark the player requested, as well as all the logic to check to see if there is a winner, end the game, and award a player a trophy if so, or to advance to the next player’s turn if not. See the [multisig_tic-tac-toe repo](https://github.com/MystenLabs/multisig_tic-tac-toe) for the full source code on this version of the game.
+([Source](https://github.com/MystenLabs/sui/blob/main/examples/tic-tac-toe/ui/src/pages/Game.tsx)).
```

Comment:
Could also use this construct to make the code available without user needing to navigate away:
```
<details>
   <summary>
      Toggle source code
   </summary>
   {@inject: examples/tic-tac-toe/ui/src/pages/Game.tsx}
</details>
```
This puts the relative path at the top, but it's not clickable.